models/pancreas_ct_dints_segmentation/docs/README.md

```diff
@@ -74,6 +73,9 @@ Users can install Graphviz for visualization of searched architectures (needed i
 
 ![Example of Searched Architecture](https://developer.download.nvidia.com/assets/Clara/Images/clara_pt_net_arch_search_segmentation_searched_arch_example_1.png)
 
+### System Memory Requirement
```

Comment:
I would move this back up to the training configuration section, here is my suggestion to match the other bundle review format: 

"### Memory Consumption
- Dataset Manager: CacheDataset
- Data Size: 420 3D Volumes
- Cache Rate: 1.0
- Multi GPU VRAM: X (8 GPUs)


### GPU Consumption Warning
If you face memory issues with CacheDataset, you can either switch to a regular Dataset class or lower the caching rate `cache_rate` in the configurations (e.g., [training](../configs/train.yaml) and [multi-GPU training](../configs/multi_gpu_train.yaml)) within range $(0, 1)$ to minimize the GPU RAM requirements."

Reply:
I've moved that part and updated the docs. Let me know if you have any other questions.